specification/storage/resource-manager/Microsoft.Storage/stable/2018-02-01/storage.json

```diff
@@ -486,7 +486,45 @@
         ],
         "responses": {
           "200": {
-            "description": "OK -- current usage acount and limit retrieved and returned successfully.",
+            "description": "OK -- current usage count and limit retrieved and returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/UsageListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Storage/locations/{location}/usages": {
+      "get": {
+        "tags": [
+          "LocationUsage"
+        ],
+        "operationId": "Usage_ListByLocation",
```

Comment:
The operation group name must be plural `Usages_ListByLocation`

Reply:
@anuchandy 

As there's already an API with name "Usage_List", to be aligned with this, we should use "Usage_ListByLocation" here. (Actually, only one usage of the specific location is returned with the API)
And this name already passed review in https://github.com/Azure/azure-rest-api-specs-pr/pull/459

Since the feature is target to 5/7 Build release, we need this to be merged in ASAP.  I will set up a meeting tomorrow to discuss this PR if it still can't be merged.

Reply:
@anuchandy 
The "Usage_List" is also aligned with other features in Usage list, like Compute: https://github.com/Azure/azure-rest-api-specs/blob/de8dadad0d5aa9f2f4b6d46a8257472a674c831e/specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json#L1006 

If we want to change “Usage” to “Usages” in all APIs, it will be a breaking of existing API "Usage_List". If only change "Usage_ListByLocation" to "Usages_ListByLocation", it will be in different operation group with "Usage_List" which is wired, since 2 APIs have similar function and same return type (only scope different: one is for current subscription, one is for a location under current subscription).

I think we might can consider to change all "Usage" to "Usages" in a future release which allows breaking change. But for this release, to be aligned with existing API and no breaking, we might should keep using "Usage_ListByLocation".

Reply:
@blueww thanks! Agree with you, to have consistently with `Storage::Usage_List` we need `Usage_ListByLocation`. We can merge this PR.

Using non-plural name for operation group is against SDK guideline, seems compute spec is also violating this guideline, thanks for pointing this. Will bring this to compute team attention and get fixed.

As you mentioned in the future API version, let's fix this. Will create an issue and assign it you.

Reply:
@blueww  Created an issue https://github.com/Azure/azure-rest-api-specs/issues/3024 and assigned to you.